### PR TITLE
Delete a few more unused imports in the old precog

### PR DESCRIPTION
**Pull request by alissapajer**

### CODE REVIEW

**Comment by alissapajer** [comment]
Arg apparently `test:compile` didn't compile the world. I'm going to rebase this and try again.

**Comment by alissapajer** [comment]
Wow is this branch in a broken state locally. It's compiling. That is deeply annoying.

File: yggdrasil/src/test/scala/quasar/yggdrasil/table/IndicesSpec.scala [hunk 1]
```diff
@@ -71,20 +71,20 @@ trait IndicesSpec extends ColumnarTableModuleTestSupport with TableModuleSpec wi
...
+      {"a": 2, "b": 2, "c": 3, "d": 1248}
+      {"a": 2, "b": 2, "c": 13}
+      {"a": "foo", "b": "bar", "c": 3}
+      {"a": 3, "b": "", "c": 333}
+      {"a": 3, "b": 2, "c": [1,2,3,4]}
+      {"a": 1, "b": 2, "c": {"cat": 13, "dog": 12}}
+      {"a": "foo", "b": 999}
+      {"b": 2, "c": 9876}
+      {"a": 1, "c": [666]}
+      """
```
Comment (edmundnoble) [inline yggdrasil/src/test/scala/quasar/yggdrasil/table/IndicesSpec.scala]: Noice.

File: yggdrasil/src/test/scala/quasar/yggdrasil/table/IndicesSpec.scala [hunk 2]
```diff
@@ -17,12 +17,12 @@
 package quasar.yggdrasil
 package table
 
+import quasar.blueeyes._, json._
```
Comment (edmundnoble) [inline yggdrasil/src/test/scala/quasar/yggdrasil/table/IndicesSpec.scala]: :+1:

**Pull request merged**